### PR TITLE
[Mellanox] Add support for SN2201 platform

### PR DESCRIPTION
**Pull request by liorghub**
Signed-off-by: liora <liora@nvidia.com>

#### Why I did it
Add support for SN2201 platform

#### How I did it
Add required content for SN2201 platform

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

File: device/mellanox/x86_64-nvidia_sn2201-r0/platform.json [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+                "1x1G[100]": ["etp48"]
+            }
+        },
+        "Ethernet48": {
+            "index": "49,49,49,49",
+            "lanes": "192,193,194,195",
+            "breakout_modes": {
+                "1x100G[50G,40G,25G,10G]": ["etp49"],
+                "2x50G[25G,10G]": ["etp49a", "etp49b"],
+                "4x25G[10G]": ["etp49a", "etp49b", "etp49c", "etp49d"]
```
Comment (liat-grozovik) [inline device/mellanox/x86_64-nvidia_sn2201-r0/platform.json:452]: for now this cannot be supported as when you do so, the other port must removed and there is no ability to support it on the dynamic port breakout. so as other systems of spectrum we don't provide the ability to do so.
Check SN2700 for example. Same behaviour.

File: device/mellanox/x86_64-nvidia_sn2201-r0/platform_components.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (liat-grozovik) [inline device/mellanox/x86_64-nvidia_sn2201-r0/platform_components.json:1]: if this is for the fw util and we dont support it yet, should we add it?

File: device/mellanox/x86_64-nvidia_sn2201-r0/platform.json (hunk 1 above)
Comment (liorghub) [reply device/mellanox/x86_64-nvidia_sn2201-r0/platform.json]: Fixed.

File: device/mellanox/x86_64-nvidia_sn2201-r0/platform_components.json (hunk 2 above)
Comment (liorghub) [reply device/mellanox/x86_64-nvidia_sn2201-r0/platform_components.json]: Done, confirmed with Nazari, file removed.

**Review by liat-grozovik — APPROVED** [review]

**Pull request closed without merging**